packages/tabs/README.md

```diff
@@ -14,7 +14,7 @@ npm install react react-dom prop-types styled-components @zendeskgarden/react-th
 
 ## Usage
 
-```jsx static
+```jsx
```

Comment:
Would it be fair to modernize the READMEs for packages moving to Storybook? Maybe it could show a hook example? 

```jsx
import { ThemeProvider } from '@zendeskgarden/react-theming';
import { Tabs, TabList, Tab, TabPanel } from '@zendeskgarden/react-tabs';

const [selectedItem, setSelectedItem] = React.useState('tab-1')

/**
 * Place a `ThemeProvider` at the root of your React application
 */
<ThemeProvider>
  <Tabs selectedItem={selectedItem} onChange={setSelectedItem}>
    <TabList>
      <Tab item="tab-1">Tab 1</Tab>
      <Tab item="tab-2">Tab 2</Tab>
      <Tab disabled>Disabled Tab</Tab>
      <Tab item="tab-3">Tab 3</Tab>
    </TabList>
    <TabPanel item="tab-1">Tab 1 content</TabPanel>
    <TabPanel item="tab-2">Tab 2 content</TabPanel>
    <TabPanel item="tab-3">Tab 3 content</TabPanel>
  </Tabs>
</ThemeProvider>
```

Reply:
This is probably as good a time as any. I'll update this to remove the styleguidist style examples